<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File_generator_ui</class>
 <widget class="QDialog" name="File_generator_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_content">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>491</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_port_generic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>port</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>generic</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>540</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>std_logic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>unsigned</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>integer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>std_ulogic</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bits">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>540</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>600</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>600</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_config">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>600</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_entity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_architecture">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_sort">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
